src/dispatch/plugins/dispatch_slack/middleware.py

```diff
@@ -112,6 +112,15 @@ def message_context_middleware(request: BoltRequest, context: BoltContext, next:
     next()
 
 
+def reaction_context_middleware(context: BoltContext, payload: dict, next: Callable) -> None:
```

Comment:
```suggestion
def reaction_context_middleware(context: BoltContext, next: Callable) -> None:
```

I think payload is unused argument.

Reply:
Good catch.